src/haddock/modules/analysis/seletop/defaults.yaml

```diff
@@ -3,8 +3,8 @@ select:
   type: integer
   min: -9999
```

Comment:
Why -9999 is an allowed value ?
Maybe information in the `long` description to explain how to use negative values could be nice ? or should it be limitted to `min = 1` ?

Reply:
I have no idea, I didn't modify it..probably makes sense to put `min=1`